python-packages/sra_client/sra_client/api/default_api.py

```diff
@@ -9,66 +9,68 @@
 import six
 
 from sra_client.api_client import ApiClient
+from sra_client.models.relayer_api_order_config_payload_schema import (
+    RelayerApiOrderConfigPayloadSchema,
+)
 
 
 class DefaultApi(object):
-    """NOTE: This class is auto generated by OpenAPI Generator
-    Ref: https://openapi-generator.tech
+    """Default API for SRA compliant 0x relayers."""
 
-    Do not edit the class manually.
-    """
+    # NOTE: This class is auto generated by OpenAPI Generator
+    # Ref: https://openapi-generator.tech
+
+    # Do not edit the class manually.
 
     def __init__(self, api_client=None):
         if api_client is None:
             api_client = ApiClient()
         self.api_client = api_client
 
-    def get_asset_pairs(self, **kwargs):  # noqa: E501
-        """get_asset_pairs  # noqa: E501
+    def get_asset_pairs(self, **kwargs):
+        """get_asset_pairs
 
-        Retrieves a list of available asset pairs and the information required to trade them (in any order). Setting only `assetDataA` or `assetDataB` returns pairs filtered by that asset only.  # noqa: E501
+        Retrieves a list of available asset pairs and the information
+        required to trade them (in any order). Setting only `assetDataA` or
+        `assetDataB` returns pairs filtered by that asset only.
         This method makes a synchronous HTTP request by default. To make an
-        asynchronous HTTP request, please pass async_req=True
-        >>> thread = api.get_asset_pairs(async_req=True)
```

Comment:
Now I'm thinking that these examples are important.  If it's too hard to get them to run as doctests that's fine, but we should keep them here, so people can understand how to do an async request with this API.  I'd (begrudgingly! :smile:) Approve a doctest skip on these examples.

Reply:
https://github.com/0xProject/0x-monorepo/pull/1734/commits/a102900483ceb28644a82f684cd0a7777b2e9b95